ChatViewController.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  ChatViewController.swift
+//  Antidote
+//
+//  Created by Dmytro Vorobiov on 02/12/16.
+//  Copyright © 2016 Dmytro Vorobiov. All rights reserved.
+//
+
+import Cocoa
+import SnapKit
+
+class ChatViewController: NSViewController {
+    fileprivate var textField: NSTextField!
+
+    override func loadView() {
+        view = NSView()
+
+        createSubviews()
+        installConstraints()
```

Comment:
In case of more complex views you can create separate class for a view and move this logic here:

```swift
class ChatView: NSView {
    ...
    func createSubviews {
        ...
    }
    
    func installConstraints {
        ...
    }
...
}
```